rustls/src/server/tls12.rs

```diff
@@ -61,6 +61,7 @@ mod client_hello {
         pub(in crate::server) session_id: SessionId,
         pub(in crate::server) suite: &'static Tls12CipherSuite,
         pub(in crate::server) using_ems: bool,
+        pub(in crate::server) force_using_ems: bool,
```

Comment:
Since `CompleteClientHelloHandling` retains access to the `config`, I don't see the value of copying `force_using_ems` in here explicitly?